kimchi/src/prover.rs

```diff
@@ -1397,9 +1397,8 @@ where
 
                     let chunks = runtime_comm
                         .blinders
-                        .chunks
-                        .iter()
-                        .map(|blinding| *joint_combiner * blinding + base_blinding)
+                        .into_iter()
```

Comment:
I am wondering if consuming the blinders here instead of iterating over the references could be a problem elsewhere 🤔

Reply:
Discussed offline.